### ISSUE TITLE
--ui does not load the correct Vitest UI app

### ISSUE THREAD

**Issue by uicrafts** [vitest-dev/vitest#833]
### Describe the bug

In version `0.5.0` in a React project, running `vitest --ui` opens this URL: http://localhost:51204/__vitest__/

However, it does not load the Vitest UI. Instead, it loads the `src/components/App.tsx` component.

Attempting to load http://localhost:51204/ without `__vitest__/` shows the same behavior.

**Additional issue** I noticed while the `vitest --ui` server was running: I updated a component, which triggered changes to an existing snapshot which would cause it to fail, and Vitest did not catch the failure. Only after I ran `yarn vitest` separately it was caught.

### Reproduction

https://stackblitz.com/edit/node-qtcaat?file=package.json

After `npm run test`, Vitest will incorrectly load the contents in  `src/components/App.tsx`.

### System Info

```shell
System:
    OS: macOS 11.6.2
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 38.23 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.0 - /usr/local/bin/node
    Yarn: 1.22.17 - ~/.yarn/bin/yarn
    npm: 8.1.2 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Browsers:
    Chrome: 98.0.4758.102
    Edge: 98.0.1108.56
    Firefox: 96.0.3
    Firefox Developer Edition: 79.0
    Safari: 15.3
    Safari Technology Preview: 15.4
  npmPackages:
    @vitejs/plugin-react: ^1.2.0 => 1.2.0
    vite: ^2.8.4 => 2.8.4
    vitest: ^0.5.0 => 0.5.0
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by uicrafts** [comment]
Just tested with `0.5.3` and this is now working as expected. I do see the time is reported in an odd way (perhaps that should be just seconds and no negative number): 
<img width="371" alt="image" src="https://user-images.githubusercontent.com/4595734/155204008-410eed82-1488-4ae5-8387-5c086bd419a4.png">

But I'll open up a separate issue for that.

**Comment by alcasas** [comment]
I'm having same behavior, I see my react app instead of the vitest ui app, any clue on this?
